.luarc.json

```diff
@@ -3,5 +3,9 @@
     "Lua.diagnostics.disable": [
         "missing-parameter"
     ],
-    "workspace.checkThirdParty": false
-}
\ No newline at end of file
+    "workspace.checkThirdParty": false,
+    "workspace.library": [
+        "../neotest",
```

Comment:
Running `nvim --headless -c ':PlenaryBustedDirectory lua/spec'` works for me without direct reference to those libraries. My understanding was that those dependencies are satisfied through plugin management.

Reply:
I was running this on new Ubuntu machine with a minimal config from [here](https://github.com/nvim-neotest/neotest-go/issues/51) and it worked without `workspace.library`

Reply:
Hapoy to remove if you like. 

To clarify, it isn't needed for running the tests for me either, as those are loaded in my vim plugin. They were needed to make the LSP aware of those plugins so that it recognized the type annotations that reference neotest types, and then the plenary/busted functions.

Reply:
removed